no longer need wordexp so remove header to fix building on uClibc

Patch by Stefan de Konink

http://bugs.gentoo.org/105478

--- src/alisp/alisp.c
+++ src/alisp/alisp.c
@@ -30,7 +30,6 @@
 #include <ctype.h>
 #include <math.h>
 #include <err.h>
-#include <wordexp.h>
 
 #define alisp_seq_iterator alisp_object
 
--- src/names.c
+++ src/names.c
@@ -31,7 +31,6 @@
  */
 
 #include <stdarg.h>
-#include <wordexp.h>
 #include <dlfcn.h>
 #include <limits.h>
 #include <sys/stat.h>
